CHANGELOG.md

```diff
@@ -13,7 +13,7 @@
 ## 3.2.0 (Oct 9, 2019)
 
 * Add support for the Batch API (@gaiottino, @teoulas)
-* Return specific exceptions for errors that might be returned from Salesforce.com - instead of getting a generic `Faraday::Error::ClientError`, you might get something like a `Restforce::EntityTooLargeError` (@boblail)
```

Comment:
Modifying changelog does seem to be a good idea :)

Reply:
Indeed..Updating.

Reply:
Done.